collagraph/renderers/pyside/objects/__init__.py

```diff
@@ -0,0 +1,9 @@
+import importlib
+from pathlib import Path
+
+
+for module in Path(__file__).parent.glob("*.py"):
```

Comment:
I'm pretty sure this doesn't work in a pyinstaller environment.

Reply:
Ai, that's right!

Reply:
Changed it back to just manually importing each module